samples/universal-react-node/src/client/components/above-the-fold.jsx

```diff
@@ -1,6 +1,5 @@
 import React from "react";
```

Comment:
leave this file alone.  We need to update above the fold support.

Rename this file to `above-the-fold.jsx.disabled` for now.

and comment out the two lines that refer to above the fold in `client/src/routes.jsx`
 
And once CI pass, we need above the fold working, else home-app can't go on react 16.